uniflow-android/src/main/java/io/uniflow/android/livedata/LiveDataPublisher.kt

```diff
@@ -27,6 +27,7 @@ open class LiveDataPublisher(
 
     override suspend fun publishState(state: UIState, pushStateUpdate: Boolean) {
         onMain(immediate = true) {
+            UniFlowLogger.logState(state)
```

Comment:
Isn't it already done on the ViewModel side?

Reply:
I'm not sure what do you mean by that, could you give more insights :D ?

Reply:
Sorry. It has been dropped somewhere. I would log event/state at `DataPublisher` side first in `LiveDataPublisher`.
Like you're doing 👍